cmd/data.go

```diff
@@ -156,8 +162,31 @@ func (cmd *DataCmd) Execute(ctx context.Context, f *flag.FlagSet, _ ...interface
 		banner = utils.GetBanner(now, dbURI)
 	} else {
 		conv.Audit.DryRun = true
+		// If migration type is Minimal Downtime, validate if required resources can be generated
+		if !conv.UI && sourceProfile.Driver == constants.MYSQL && sourceProfile.Ty == profiles.SourceProfileTypeConfig && sourceProfile.Config.ConfigType == constants.DATAFLOW_MIGRATION {
+			spClient, err:= spinstanceadmin.NewInstanceAdminClientImpl(ctx)
+			if err != nil {
+				return subcommands.ExitFailure
+			}
+			dsClient, err := datastreamclient.NewDatastreamClientImpl(ctx)
+			if err != nil {
+				return subcommands.ExitFailure
+			}
+			storageclient, err := storageclient.NewStorageClientImpl(ctx)
+			if err != nil {
+				return subcommands.ExitFailure
+			}
```

Comment:
If any of these fails, what error message would be shown to the user? 
We should log the error if this would fail silently.